pages/configuring_webif.rst

```diff
@@ -308,106 +308,127 @@ If you want to run a load balancer/reverse proxy in front of Graylog, you need t
-    }
-  }
+**REST API and Web Interface on one port (using HTTPS/SSL)**::
 
-  server
-  {
-    listen      12900 ssl spdy;
-    server_name graylog.example.org;
+    server
+    {
+        listen      443 ssl spdy;
+        server_name graylog.example.org;
+        # <- your SSL Settings here!
 
     location /
-    {
-        proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header    Host $http_host;
-        proxy_pass          http://graylog.internal.example.org:12900/;
+        {
+            proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header    Host $http_host;
+            proxy_set_header    X-Graylog-Server-URL https://graylog.example.org/api;
+            proxy_pass          http://127.0.0.1:9000;
+        }
+    location /api/
+        {
+            proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header    Host $http_host;
+            proxy_pass          http://127.0.0.1:12900/;
+        }
     }
-  }
 
 Apache:
 -------
 
-For the following samples we are assuming that your Graylog instance is running on ``graylog.internal.example.org`` using the default ports of 12900 for the REST API and 9000 for the web interface. Apache is running on the same server as Graylog. SSL is disabled for both. You want to expose the Graylog web interface as ``https://graylog.example.org``.
-
-**URI Configs in Graylog server conf**::
-
-   rest_listen_uri = http://127.0.0.1:12900/
-   web_listen_uri = http://127.0.0.1:9000/
-
-
-**REST API and Web Interface on one port (using HTTPS/SSL)**::
-
-   Listen 443
-   <VirtualHost *:443>
-       ServerName graylog.example.org
-       # Your SSL config <-- You should change this
-       <Location /api/>
-           ProxyPass http://127.0.0.1:12900/
-           ProxyPassReverse http://127.0.0.1:12900/
-       </Location>
-       <Location />
-           RequestHeader set X-Graylog-Server-URL "https://graylog.example.org/api/"
-           ProxyPass http://127.0.0.1:9000/
-           ProxyPassReverse http://127.0.0.1:9000/
-       </Location>
-   </VirtualHost>
-
-.. warning:: Using Apache 2.2 needs the configuration above, if you have Apache 2.4 you need to switch the Locations. This means ``/api/`` must go after ``/``
+**REST API and Web Interface on one port (using HTTP)**::
+
+	<VirtualHost *:80>
+		ServerName graylog.example.org
+		ProxyRequests Off
+		<Proxy *>
+			Order deny,allow
+			Allow from all
+		</Proxy>
+		<Location /api/>
+			ProxyPass http://127.0.0.1:12900/
+			ProxyPassReverse http://127.0.0.1:12900/
+		</Location>
+		<Location />
+			RequestHeader set X-Graylog-Server-URL "http://graylog.example.org/api/"
+			ProxyPass http://127.0.0.1:9000/
+			ProxyPassReverse http://127.0.0.1:9000/
+		</Location>
+	</VirtualHost>
+
+.. CAUTION:: Using Apache 2.2 needs the configuration above, if you have Apache 2.4 you need to switch the Locations. This means ``/api/`` must go after ``/``
```

Comment:
Maybe it would be better to add a written out and copy & pastable version for both Apache httpd versions instead of adding this warning. I'm not sure if many people would read that far. :wink: